From 52ecc54a0a0b7abd8f39abe9a56e7be03f4e4c84 Mon Sep 17 00:00:00 2001
From: Jakob Bornecrantz <jakob@vmware.com>
Date: Mon, 14 Dec 2009 22:07:45 +0000
Patch-mainline: 2.6.33
References: fate#310916
Git-commit: be1cb8689c480228ffd2e4bfccc0dab7156cd9ea
Subject: [PATCH 0189/2588] drm/ttm: Add more driver type enums

Signed-off-by: Jakob Bornecrantz <jakob@vmware.com>
Signed-off-by: Dave Airlie <airlied@redhat.com>
(cherry picked from commit be1cb8689c480228ffd2e4bfccc0dab7156cd9ea)

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 include/drm/ttm/ttm_object.h |    6 +++++-
 1 files changed, 5 insertions(+), 1 deletions(-)

diff --git a/include/drm/ttm/ttm_object.h b/include/drm/ttm/ttm_object.h
index 703ca4d..0d9db09 100644
--- a/include/drm/ttm/ttm_object.h
+++ b/include/drm/ttm/ttm_object.h
@@ -77,7 +77,11 @@ enum ttm_object_type {
 	ttm_buffer_type,
 	ttm_lock_type,
 	ttm_driver_type0 = 256,
-	ttm_driver_type1
+	ttm_driver_type1,
+	ttm_driver_type2,
+	ttm_driver_type3,
+	ttm_driver_type4,
+	ttm_driver_type5
 };
 
 struct ttm_object_file;
-- 
1.7.6

